CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.5.1 (Apr 20, 2022). Tested on Artifactory 7.37.14
+
+BUG FIXES:
+
+* provider:  Setting the right default value for 'access_token' attribute. [GH-426]
+
 ## 6.5.0 (Apr 15, 2022). Tested on Artifactory 7.37.14
```

Comment:
hm... this date for 6.5.0 seems wrong too, wanna fix that too while you're here?